dirsrvtests/tests/suites/replication/regression_m2_test.py

```diff
@@ -1218,6 +1218,9 @@ def test_rid_starting_with_0(topo_m2, request):
     for replica,rid in zip(replicas, ['010', '020']):
         replica.replace('nsDS5ReplicaId', rid)
 
+    S1.restart()
```

Comment:
Why restarting the instance is needed ?
 ( If it is really needed it is worth to explain why in  a comment )